OTKit/otkit-icons/ic_checkmark.svg

```diff
@@ -3,13 +3,7 @@
     <!-- Generator: Sketch 52.5 (67469) - http://www.bohemiancoding.com/sketch -->
     <title>icon/line/ic_checkmark/Light UI - Default</title>
     <desc>Created with Sketch.</desc>
-    <defs>
+    <g id="icon/line/ic_checkmark/Light-UI---Default" stroke="none" stroke-width="1" fill="#2D333F" fill-rule="evenodd">
```

Comment:
added `id`?

Reply:
SVGO will clean up the ids.  if you look at the full file diff, im basically removing the `defs` and replacing them with the `g` that was below in the file. the id isn't added, just moved

Reply:
if this is part of your PR, can you just remove them?